docs/installation/fedora.md

```diff
@@ -123,13 +123,19 @@ There are two ways to install Docker Engine.  You can install with the `yum` pac
 
 4. Start the Docker daemon.
 
-		$ sudo service docker start
+    On Fedora < 21
```

Comment:
We actually no longer support Fedora < 21, so I think it's good to just change this line